client/my-sites/themes/theme-showcase.jsx

```diff
@@ -138,6 +129,47 @@ class ThemeShowcase extends Component {
 		}
 	}
 
+	getTabTiers = ( { translate } ) => {
+		return [
+			{ value: 'all', label: translate( 'All' ) },
+			{ value: 'free', label: translate( 'Free' ) },
+			{ value: 'premium', label: translate( 'Premium' ) },
+		];
+	};
+
+	getTabFilters = ( { subjects = {}, translate } ) => {
+		const subjectFilters = Object.fromEntries(
+			shuffle(
```

Comment:
Unfortunately, the entire component is re-mount when coming back from theme details. I've removed the randomization for now, since I couldn't find a clean way to do it without perhaps creating a new function in `client/lib`. Since it's not part of spec, we can do this later. What do you think?

Reply:
Sounds good 👍